packages/coinstac-ui/scripts/build.js

```diff
@@ -22,14 +22,24 @@ buildNative()
   return packager(options);
 })
 .then((appPath) => {
-  console.log(`Finished building at: ${appPath}`); // eslint-disable-line no-console
-  console.log('Now tarring...'); // eslint-disable-line no-console
+  let zip;
   const dir = `${__dirname}/../coinstac-${os.platform()}-${os.arch()}`;
 
-  const read = zip().createReadStream(`${dir}`);
-  const write = fs.createWriteStream(`${dir}.tar.gz`);
+  console.log(`Finished building at: ${appPath}`); // eslint-disable-line no-console
+  console.log('Now archiving...'); // eslint-disable-line no-console
+
+  if (os.platform() === 'win32') {
+    zip = archiver.create('zip');
+  } else {
+    zip = archiver.create('tar', { gzip: true });
+  }
+  const write = fs.createWriteStream(os.platform() === 'win32' ? `${dir}.zip` : `${dir}.tar.gz`);
```

Comment:
Minor issue, but you’re branching on `os.platform()` twice. Want to move that all into one spot? Or, write `zip` declaration as:

```js
const zip = os.platform() === 'win32' ? archiver.create('zip') : archiver.create('tar', { gzip: true });
```

Would make it a little easier to read.